src/risedevtool/src/config.rs

```diff
@@ -50,6 +50,24 @@ impl ConfigExpander {
 
     /// Transforms `risedev.yml` and `risedev-profiles.user.yml` to a fully expanded yaml file.
     ///
+    /// Format:
+    ///
+    /// ```yaml
+    /// my-profile:
+    ///   config-path: src/config/ci-recovery.toml
+    ///   envs:
+    ///     - "RUST_LOG=info,risingwave_storage::hummock=off"
+    ///     - "RW_ENABLE_PRETTY_LOG=true"
```

Comment:
Shall we make it a map, like

https://github.com/risingwavelabs/risingwave/blob/5ea0b67dfc2639800c8fb76abfaa5fc8138e0f0e/.github/workflows/dashboard.yml#L33-L38